### ISSUE TITLE
"File has no contents" when file is in a directory with spaces in directory name

### ISSUE THREAD

**Issue by dstansby** [atom/github#1235]
### Description
Trying to view the diff of a file that is in a directory with spaces in the filename (ie. "my directory/myfile.py") appears with the "File has no contents" message instead of displaying the diff.

Renaming the directory to not have a space fixes the issue (ie. "mydirectory/myfile.py")

### Steps to Reproduce

1. Commit a file in a directory with a space in its name
2. Change something in the file
3. Try to open the file's diff

**Expected behavior:**

Diff is shown as normal

**Actual behavior:**
"File has no contents" message shown and diff not shown

**Reproduces how often:**

All the time

### Versions

```
$ atom --version
Atom    : 1.21.1
Electron: 1.6.15
Chrome  : 56.0.2924.87
Node    : 7.4.0
```

```
$ apm --version
apm  1.18.5
npm  3.10.10
node 6.9.5 x64
python 3.6.3
git 2.14.1
```

MacOS 10.13

**Comment by 50Wliu** [comment]
Fixed by #1217.

**Comment by dstansby** [comment]
Woops, didn't check the closed issues, thanks!
